source/educators/references/resources_for_course_teams.rst

```diff
@@ -1,58 +1,48 @@
-**********************************
-Resources for edx.org Course Teams
-**********************************
+###################################
+Resources for Open edX Course Teams
+###################################
 
 .. tags:: educator, reference
 
 Course teams include faculty, instructional designers, course staff, discussion
 moderators, and others who contribute to the creation and delivery of courses
-on edx.org or edX Edge.
-
-======================================
-The edX Course Creator Series
-======================================
-
-The courses in the edX Course Creator Series provide foundational knowledge
-about using the edX platform to deliver educational experiences. These courses
-are available on edx.org.
+on an Open edX site.
 
 .. contents::
  :local:
  :depth: 1
 
-edX101: Overview of Creating a Course
-**************************************
+Open edX Demo Course
+====================
 
-The `edX101`_ course is designed to provide a high-level overview of the course
-creation and delivery process using Studio and the edX LMS. It also highlights
-the extensive capabilities of the edX platform.
+The `Open edX Demo Course <https://sandbox.openedx.org/courses/course-v1:OpenedX+01-2024+2024-1/about>`_ course is designed to In discuss the tools and features that can leverage your learning program. After providing details about the platform, the course shows the different types of content that can be created, showcase the variety of assessment tools, and explore the different ways that you can engage your learners with social interactions.
 
-StudioX: Creating a Course with edX Studio
-*************************************************
+StudioX: Creating a Course with Open edX Studio
+===============================================
 
-After you complete edX101, "StudioX" provides more detail about using Studio
+After you complete Open edX101, "StudioX" provides more detail about using Studio
 to create a course, add different types of content, and configure your course
 to provide an optimal online learning experience.
 
-BlendedX: Blended Learning with edX
-************************************
+BlendedX: Blended Learning with Open edX
+========================================
 
 In "BlendedX" you explore ways to blend educational technology with
 traditional classroom learning to improve educational outcomes.
 
-VideoX: Creating Video for the edX Platform
-*************************************************
+VideoX: Creating Video for the Open edX Platform
```

Comment:
Please remove this, there is no Video X course

Reply:
done :)